eng/dockerfile-templates/sdk/Dockerfile

```diff
@@ -76,11 +77,6 @@ RUN `
     && powershell Remove-Item -Force -Recurse "%TEMP%\*" `
     && rmdir /S /Q "%ProgramData%\Package Cache"
 
-# Install web targets
-RUN curl -fSLo MSBuild.Microsoft.VisualStudio.Web.targets.zip {{VARIABLES["vs|webTargetsUrl"]}} `
```

Comment:
Is the webTargetsUrl variable needed anymore in the manifest.versions.json?  Can this get cleaned up?

Reply:
Yep, thanks for pointing that out.